drivers/sensor/sht3xd/sht3xd.c

```diff
@@ -43,22 +44,8 @@ static const int measure_wait[3] = {
 static uint8_t sht3xd_compute_crc(uint16_t value)
 {
 	uint8_t buf[2] = { value >> 8, value & 0xFF };
```

Comment:
Done.
As you suggested to squash the "crc8 commit" for the sht3xd driver, I have done this refactoring also in the same commit.
But as this is not dependent on the new driver it would probably be better to do this refactoring in a separate commit,
wouldn't it?